povm_toolbox/library/mutually_unbiased_bases_povm.py

```diff
@@ -0,0 +1,126 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""TODO."""
+
+from __future__ import annotations
+
+import numpy as np
+from numpy.random import Generator
+from scipy.spatial.transform import Rotation
+
+from .randomized_projective_measurements import RandomizedProjectiveMeasurements
+
+
+class MutuallyUnbiasedBasesPOVM(RandomizedProjectiveMeasurements):
+    """A mutually-unbiased-bases (MUB) POVM."""
+
+    def __init__(
+        self,
+        n_qubit: int,
+        bias: np.ndarray,
+        angles: np.ndarray,
+        measurement_twirl: bool = False,
+        measurement_layout: list[int] | None = None,  # TODO: add | Layout
+        shot_repetitions: int = 1,
+        seed_rng: int | Generator | None = None,
+    ):
+        """Implement a mutually-unbiased-bases (MUB) POVM.
+
+        This is a special case of a :class:`.RandomizedProjectiveMeasurements`. A MUB POVM corresponds
+        to an arbitrary rotated :class:`.LocallyBiasedClassicalShadows`.
```

Comment:
Maybe you can be a bit more specific about how the MUB lies between the LBCS and RPM.
Especially, since the RPM also has an `angles` argument but which behaves differently. I think it is important that the exact difference between these two is explained very well.